app/src/main/java/com/infomaniak/mail/data/cache/mailboxContent/DraftController.kt

```diff
@@ -148,11 +149,14 @@ class DraftController @Inject constructor(
             return@let document.outerHtml()
         } ?: ""
 
+        val subBodiesContent = message.body?.subBodies?.let(UiUtils::formatSubBodiesContent) ?: ""
+
         return """
             <div id=\"answerContentMessage\" class="${MessageBodyUtils.INFOMANIAK_REPLY_QUOTE_HTML_CLASS_NAME}" >
             <div>$messageReplyHeader</div>
             <blockquote class=\"ws-ng-quote\">
             $previousBody
+            $subBodiesContent
```

Comment:
Is this where we want to insert subbodies?

Reply:
Yes.

Reply:
Yes?